pkg/meta/model/job_args.go

```diff
@@ -1291,6 +1291,18 @@ func GetAddCheckConstraintArgs(job *Job) (*AddCheckConstraintArgs, error) {
 	return getOrDecodeArgsV2[*AddCheckConstraintArgs](job)
 }
 
+// KeyRange is copied from kv.KeyRange to avoid cycle import.
+type KeyRange struct {
+	StartKey []byte `json:"start_key,omitempty"`
+	EndKey   []byte `json:"end_key,omitempty"`
```

Comment:
keep json tag the same to be compatible

Reply:
Remove the json tags as `kv.KeyRange` does?

Reply:
yes